support-frontend/assets/pages/[countryGroupId]/components/checkoutComponent.tsx

```diff
@@ -543,6 +541,10 @@ export function CheckoutComponent({
 			? 'ANNUAL'
 			: 'ONE_OFF';
 
+	/*
+  TODO :  Passed down because minimum product prices are unavailable in the paymentTsAndCs story
+          We should revisit this and see if we can remove this prop, pushing it lower down the tree
+  */
 	const thresholdAmount = getLowerProductBenefitThreshold(
```

Comment:
`thresholdAmount` will be revisited in a later PR but remains a workaround prop here